baton/hub.go

```diff
@@ -40,45 +40,57 @@ func (h hub) run() {
 				close(c.send)
 			}
 		case rawMsg := <-h.receive:
-			fmt.Println(rawMsg)
 			var cmd command
 			if err := json.Unmarshal(rawMsg.data, &cmd); err != nil {
 				fmt.Println("nu")
```

Comment:
Yeah, we need to cast the []byte slice to string to get readable output, as so:

```go
fmt.Println(string(rawMsg))
```

Should we just go ahead and remove it for now?